src/allegro_flare/platform/linux/clipboard_linux.cpp

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
Yes, can you submit `clipboard_linux` and the renaming of `platform_generic` as a separate PR?  As it looks I would merge it right away.

Reply:
Yep, I'll do that. I will also actually implement a shared clipboard for Linux (after delving in some Xorg and Wayland doc)

Reply:
> I will also actually implement a shared clipboard for Linux (after delving in some Xorg and Wayland doc)

Very cool.  I was looking into https://github.com/dacap/clip or another 3rd-party solution as a way to support all the different implementations.  I didn't get too far, but maybe something similar could work for the underlying grunt work.